README.md

```diff
@@ -17,8 +17,9 @@ SQL DSLs by:
 - Supporting mandatory and optional named parameters.
 - Allowing query composability with ease.
 - Working out of the box with PostgreSQL using
-  [Ecto](https://github.com/elixir-ecto/ecto_sql) or
```

Comment:
Maybe this whole PR can be summarized by adding the following to the docs:

```markdown
- Working out-of-the-box with PostgreSQL using
  [Ecto](https://github.com/elixir-ecto/ecto_sql) or
  [Postgrex](https://github.com/elixir-ecto/postgrex).
- Working out-of-the-box with any database that has Ecto support
  e.g. MySQL via [MyXQL](https://github.com/elixir-ecto/myxql).
```

Reply:
hecho!